builders/tools/python.py

```diff
@@ -31,7 +31,7 @@ def install(dir_name,version=None):
             # Assumes no python2 version is installed
             v = LooseVersion(version)
             if v.version[0] == 3:
-                version_short = v.version[:2]
+                version_short = ".".join(map(str, v.version[:2]))

```

Comment:
Not sure if it's more readable as
```python
'.'.join(str(x) for x in v.version[:2])
```
I don't have a strong opinion on this. What do you think?